src/runtime/c_runtime_api.cc

```diff
@@ -450,6 +464,31 @@ int TVMBackendFreeWorkspace(int device_type, int device_id, void* ptr) {
   return 0;
 }
 
+void* TVMBackendAllocTexture(int device_type, int device_id, uint64_t width, uint64_t height,
+                             int dtype_code_hint, int dtype_bits_hint) {
+  TVMContext ctx;
+  ctx.device_type = static_cast<DLDeviceType>(device_type);
```

Comment:
I wonder if it is possible to blend into a PackedFunc now so it can be OpenCL specific